JavaScript_Course_Code/JS/4_my _js_example.js

```diff
@@ -0,0 +1,10 @@
+function greeting(){
+    var Myname;
+    Myname = prompt("Enter your name");
+    document.getElementById("name").innerHTML = Myname;
+}
+
+function changeColor (mycolor){
+    const ele = document.getElementById("name");
+    ele.style.color=mycolor;
+}
```

Comment:
This function was I think copied from MDN docs. But no problem. Its good that you are learning.